app.js

```diff
@@ -0,0 +1,183 @@
+'use strict'
+
+var = pikeStore {  // 1st and Pike Store
+  minCust: 23;
+  maxCust: 65;
+  avgCookie: 6.3;
+  totalSales: 0;
+  hourSales: [];
+
+  storeRandCust: function() {  // random number
+    return Math.random() * (this.maxCust - this.minCust) + this.minCust;
```

Comment:
this equation is super close...checkout the math.random docs for an example of one that we need to use (hint: you have to add '1' in there)